packages/docmanager-extension/src/index.ts

```diff
@@ -206,6 +229,42 @@ function addCommands(app: JupyterLab, docManager: IDocumentManager, palette: ICo
     execute: () => { app.shell.closeAll(); }
   });
 
+  commands.addCommand(CommandIDs.closeOtherTabs, {
+    label: () => `Close Other Tabs`,
+    isEnabled: () => {
+      const widgets = toArray(app.shell.widgets('main'));
```

Comment:
I wanted to check that at least two widgets exists, otherwise there is no need to show the Close Others option since there are no other tabs.  I had originally used the iterator thinking it was more efficient but I didn't like the way it looked so I decided to use code which was clearer on the intent even if it wasn't as efficient.

I can use the iterator if you think that is best.  I just need to call next() twice to ensure there are two widgets.